jac-uk/apply
# #1021 Barrister - Pupilage Exemption Manage Upload

## What's included?
Updated this question from Why you were exempt from pupillage (we may ask for a copy of your exemption or practicing certificate)? to Why you were exempt from pupillage?
Added an optional question - Please provide a copy of your exemption and or practicing certificate which should only appear if 'barrister' has been selected and 'Have you completed pupillage' is set to no. These file uploads will appear at the bottom of the page as they should only appear once in the page no matter how many sections are added.

This PR is linked to the associated admin changes in the following PR: https://github.com/jac-uk/admin/pull/2030

Closes #1021 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Apply for a new role
- Click on 'Relevant Qualifications'
- Select 'Barrister'
- Select "No" to completing pupillage
- Ensure the option to upload an Exemption Certificate and Practicing Certificate are available at the bottom of the form
- If the form is submitted (without uploading any files) you get errors to tell you that you must upload at least one of these files
- If you open another section and select 'Barrister' you will still only see one set of file upload fields for Exemption Certificate and Practicing Certificate
- If you upload one or more files then press Save and Continue the error disappears
- If you go back to the page the file name of the file you uploaded appears next to the file upload form
- You can overwrite the uploaded file with another file

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
